shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -313,7 +421,7 @@ \subsection{New Epoch Transition}
 Figure~\ref{fig:rules:new-epoch} defines the new epoch state transition. It has three rules.
 The first rule describes the change in the case of $e$ being equal to the next epoch
 $e_\ell+ 1$. It also calls the $\mathsf{EPOCH}$ rule and checks that the
-instantaneous rewards stored in the reward update is the same as the instantaneous rewards 
+instantaneous rewards stored in the reward update is the same as the instantaneous rewards
```

Comment:
is this still true now?

Reply:
nope! there is no `i_rwd`s in the reward update, good catch!

Reply:
I've reworded it.